scripts/compile_tests/failures_histogram.py

```diff
@@ -73,21 +56,32 @@ def repro(testcase):
 
 
 # e.g. "17c5f69852/dynamo"
-def failures_histogram(directory):
-    xmls = open_test_results(directory)
-    dct = get_failures(xmls)
+def failures_histogram(eager_dir, dynamo_dir):
+    fail_keys = compute_pass_rate(eager_dir, dynamo_dir)
+    xmls = open_test_results(dynamo_dir)
+
+    testcases = get_testcases(xmls)
+    testcases = [t for t in testcases if key(t) in fail_keys]
+    dct = get_failures(testcases)
 
     a = [(x, y, repro(z[0])) for x, y, z in dct]
 
+    counts, _, _ = zip(*a)
+
+    print("(num_failed_tests, error_msg, sample_test)")
     for row in a:
         print(row)
+    print("[counts]", sum(counts))
 
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
         prog="failures_histogram",
         description="See statistics about skipped Dynamo tests",
     )
-    parser.add_argument("directory")
+    # linux-focal-py3.11-clang10 (default) Test Reports (xml) directory
+    parser.add_argument("eager_dir")
```

Comment:
It may be helpful to add `help=""` with the comment too so that failures_histogram.py --help works